test_stack.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import pytest
+from stack import Stack
+
+
+def test_initialization():
+    stack = Stack()
+    assert isinstance(stack, Stack)
+    assert stack.list.head is None
```

Comment:
The first assertion is not particularly useful.  If the `Stack` class' `__init__` method doesn't return a `Stack` instance, something is quite wrong.

The second assertions is much more useful.  It verifies that `__init__` behaves as expected with regard to how it sets up the list contained in the stack.  Good.